redis/examples/tls.rs

```diff
@@ -0,0 +1,44 @@
+use redis::{Commands, ConnectionAddr, ConnectionInfo, RedisConnectionInfo};
```

Comment:
I'd prefer not to add yet another example for this.

Reply:
Removed. Is there some documentation you would like me to update?

Reply:
Yeah, whatever the public entry point is (which I don't quite recall) would probably be good to mention what this is, what it expects to get, that sort of thing.